extensions/mailer/runtime/src/main/java/io/quarkus/mailer/impl/MailConfig.java

```diff
@@ -0,0 +1,129 @@
+package io.quarkus.mailer.impl;
+
+import java.util.Optional;
+import java.util.OptionalInt;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "mailer", phase = ConfigPhase.RUN_TIME)
+public class MailConfig {
+
+    /**
+     * Configure the default `from` attribute.
+     * It's the sender email address.
+     */
+    @ConfigItem
+    public Optional<String> from;
+
+    /**
+     * Enables the mock mode, not sending emails.
+     * The content of the emails is printed on the console.
+     */
+    @ConfigItem
+    public Optional<Boolean> mock;
```

Comment:
It's better to just use a primitive `boolean` with a default value than having a tristate like this.

Reply:
Done.